control/passivity.py

```diff
@@ -4,7 +4,7 @@
 '''
 
 import numpy as np
-from control import statesp as ss
+from control import statesp
```

Comment:
The name collision was a friendly reminder, that you should only export your own functions when `from .passivity import *` is used: Define `__all__`.